### PR TITLE
docs/generic workflow

### PR DESCRIPTION
**Pull request by Jfoul**
Adding a generic workflow of setting an eCalc model up (at some point to be accompanied by an example).

### CODE REVIEW

File: docs/docs/about/modelling/workflow/generic_workflow.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+#### Additional Information
+
+Any emissions that do not fall within the defined categories can still be considered for a given platform. For example, if there are drilling activities, an additional fuel type can be specified and related to the fuel consumption of a drilling rig. 
+
+### Consumer Information
+
+#### Generator Set
+
+As eCalc™ does not indepthly model gas turbine generators, alternative methods are used. 
+Here, fuel consumed and electricity generated is related in tabular form. These are typically linear relationships, and if more than one generator is used, "generator switching" is modelled by adding another generator curve on top of the existing.
```
Comment (HeddaSvendsen) [inline docs/docs/about/modelling/workflow/generic_workflow.md:136]:
```suggestion
Here, fuel consumed and power generated is related in tabular form. These are typically linear relationships, and if more than one generator is used, "generator switching" is modelled by adding another generator curve on top of the existing.
```

**Review by HeddaSvendsen — APPROVED** [review]
Great work Jason!

Just a few small improvement suggestions from me.

Also, since this is a generic workflow not only for synthetic models, I would be a bit cautious to state that they should adjust the charts. Sometimes other solutions like POWERLOSSFACTOR could be realistic and good enough, and it could also be invalid for example due to higher rates than the system can handle (i.e. they need to adjust the rates or make changes to their equipment). I do not have a good suggestion, but maybe use "consider to adjust" as a temporary solution.

**Comment by HeddaSvendsen** [comment]
@frodemar Can you have a look when you have time?

File: docs/docs/about/modelling/workflow/generic_workflow.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jfoul) [reply docs/docs/about/modelling/workflow/generic_workflow.md:136]: yes, I can just use power :)

**Comment by Jfoul** [comment]
I think if the points are invalid, there is no reason to mention the powerlossfactor (but if they do not correlate, i.e. historical v measured), then I can mention the powerlossfactor. 

Such like:
image

**Pull request merged**